examples/simulated/plot_classifier_comparison.py

```diff
@@ -85,11 +85,12 @@ def get_proba(cov_00, cov_01, cov_11, clf):
         y
     ),
     make_gaussian_blobs(
-        2*n_matrices, n_channels, random_state=rs, class_sep=1., class_disp=.2
+        2*n_matrices, n_channels, random_state=rs, class_sep=1., class_disp=.2,
+        n_jobs=-1
```

Comment:
don't do this. If someone runs this code a big shared computer without paying attention it will be mess